README.markdown

```diff
@@ -5,8 +5,15 @@ for the 6model object system. It is primarily aimed at running NQP and Rakudo
 Perl 6, but should be able to serve as a backend for any compilers built using
 the NQP compiler toolchain.
 
-## Build It
+## Get It
+Either download it from https://www.moarvm.org/ or clone it from GitHub:
```

Comment:
Can we make that a a real link, like `[the MoarVM site](https://www.moarvm.org/)`? Thanks!